src/main.cpp

```diff
@@ -1299,8 +1299,7 @@ bool VerifyAmounts(const CCoinsViewCache& cache, const CTransaction& tx, std::ve
                 if (!MoneyRange(val.GetAmount()))
                     return false;
 
-                if (val.GetAmount() == 0)
-                    continue;
+                assert(val.GetAmount() != 0);
```

Comment:
Just to clarify, does secp256k1_pedersen_commit require val.GetAmount() > 0 or is this an emergency assert?